src/Process/Listener/Mutex/Redis.php

```diff
@@ -17,7 +17,16 @@ class Redis extends ListenerAbstract implements RedisInterface
     protected function _run(): Forked
     {
         $this->_getProcessSignal()->setCanBufferSignals(false);
-        $this->_register();
+        try {
+            $this->_register();
+        } catch (\Throwable $throwable) {
+            posix_kill($this->getParentProcessId(), $this->getParentProcessTerminationSignalNumber());
```

Comment:
Oh neat, I didn't know there were different termination signals for different process types